app/components/Sidebar.js

```diff
@@ -20,7 +20,7 @@ function Sidebar() {
   }, [userInfo]);
 
   if (!userInfo) {
-    return null; // Zeige nichts, wenn der Benutzer nicht eingeloggt ist
+    return <div>Benutzerinformationen werden geladen...</div>;
```

Comment:
Wäre es hier nicht vllt sinnvoll, wenn einfach ein Placeholder geladen wird, der vielleicht nur unten den User enthält? Dann hat man einen schöneren wechsel?
Für die Items könnten Skeletons verwenden werden: https://mui.com/material-ui/react-skeleton/

Ansonsten siehts gut aus

Reply:
skeletons können wir im nachhinein mit einem neuen Pullrequest machen. 
die änderung hier ist ja nur damit die sidebar nicht verschwindet wenn man die seite aktualisiert. 
die nachricht kommt nur ganz kurz, 
kann ich auch weg lassen wenn ihr wollt

Reply:
änderung angepasst. Sidebar gibt beim laden keine nachricht mehr. 
Vorteil vom localstorage. 
Wenn ein Nutzer angemeldet ist und während dessen sich die Rolle wechselt, ändert es sich in Echtzeit auch im Sidebar